index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Brian and Michael Personal Portfolio webpage</title>
+    <link rel="stylesheet" href="styles.css"/>
+  </head>
+  <body>
+    <nav id="navbar">
+      <ul id="nav-list">
+        <li>
+          <a href="welcome-section">Information<a/>
+        </li>
+        <li>
+          <a href="#projects">Activities<a/>
+        </li>
+        <li>
+          <a href="#contact">Contact us<a/>
+        </li>
+      </ul>
+    </nav>
+    <section id="welcome-section" class="welcome-section">
+      <h1>Hey, We are Brian and Michael</h1>
+      <p>We are upcoming Web Developers</p>
+    </section>
+    <section id="projects" class="projects">
+      <h2>We are currently working on this Project</h2>
+        <div class="project-grid">
+          <a href="">
+            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8d/President_Barack_Obama.jpg/1200px-President_Barack_Obama.jpg" alt="" class="project-img"/>
+            <p class="project-tile">The Story of President Barrack Obama</p>
+          </a>
+          <a href="">
+            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/d/d7/US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg/1200px-US_President_Barack_Obama_taking_his_Oath_of_Office_-_2009Jan20.jpg" alt="" class="project-img"/>
```

Comment:
- [ ] Please ensure that each of your image tags has an alt attribute. Alt attributes are important, especially for visually impaired people who depend on screen readers /to access your webpage.